spark-3.1-spanner-lib/src/main/java/com/google/cloud/spark/spanner/SpannerTable.java

```diff
@@ -93,6 +93,9 @@ public StructType createSchema(String tableName, ResultSet rs) {
   }
 
   public static DataType ofSpannerStrType(String spannerStrType, boolean isNullable) {
+    // Trim both ends of the string firstly, it could have come in as:
+    // "   STRUCT<a STRING(10), b INT64>  "
+    spannerStrType = spannerStrType.trim();
```

Comment:
Please add `.toUpperCase()` to make sure the switch won't fail

Reply:
Done and thanks David!